config/routes.rb

```diff
@@ -3,13 +3,21 @@
   ActiveAdmin.routes(self)
   root "posts#index"
 
-  get "posts/delete", to: "posts#delete"
-  get "posts/confirm", to: "posts#confirm"
+  # Can we remove these in favor of using the flash, and redirecting?
```

Comment:
Probably. I think I tried once and there was an issue that made it more pain that it was worth at the time.